src/scroll-timeline-css-parser.js

```diff
@@ -462,7 +505,11 @@ export class StyleParser {
 
     const anonymousMatch = RegexMatcher.ANONYMOUS_SCROLL.exec(shorthand);
     if(!anonymousMatch) {
-      timelineName = this.findMatchingEntryInContainer(shorthand, this.scrollTimelineOptions);
+      timelineName =
```

Comment:
Nit indentation looks off.

new Set(...) should have the same indent  as shorthand here and below.

timeline = 
....this.find...(
........shorthand,
........new Set(...),
....this.find...(
........shorthand,
........new Set(...)));

Reply:
Looks like you are using indent 2 in a linebroken statement.  Should be indent 4 per the Google style guide.